frontends/election-manager/src/components/smartcard_modal/status_message.tsx

```diff
@@ -74,7 +74,9 @@ export function StatusMessage({
         throwIllegalValue(action);
       }
     }
-    return <Modal content={<Loading as="p">{text}</Loading>} />;
+    return (
+      <Modal centerContent content={<Loading as="strong">{text}</Loading>} />
```

Comment:
As noted in the commit message, I'm holding off of making in-progress text larger until we switch to new
user role names, which are longer. I'll test larger text then